### PR TITLE
add documentation for --output-file cli option. fixes #4319

### LINKED ISSUE #4319: Output-File option is missing from CLI docs
> Clearly describe the documentation issue

The docs are missing the --output-file / -o option added in #4085 / #3586 

> What solution would you like to see?

I suggest adding an explanation of that option to the docs and the README.md

### PR DESCRIPTION
**Pull request by bkinnamon**
> Which issue, if any, is this issue related to?

Closes #4319 

> Is there anything in the PR that needs further explanation?

No, it's self explanatory

### CODE REVIEW

**Comment by bkinnamon** [comment]
I tried my best to match the style of the rest of the docs. If anything needs to be changed, please let me know.

**Comment by vankop** [comment]
`coveralls` is down again..

**Comment by evilebottnawi** [comment]
@vankop i think we can ignore this

File: docs/user-guide/cli.md [hunk 1]
```diff
@@ -96,6 +96,16 @@ If you're using both these special comments and autofixing, please run stylelint
 
 For CSS with standard syntax, stylelint will use [postcss-safe-parser](https://github.com/postcss/postcss-safe-parser) to fix syntax errors.
 
+### Write Report to a File
+
+With the `--output-file filename` or `-o filename` option, stylelint will output the report to the specified `filename` in addition to the standard output.
```
Comment (hudochenkov) [inline docs/user-guide/cli.md]:
```suggestion
With the `--output-file filename` option, stylelint will output the report to the specified `filename` in addition to the standard output.
```

I think we don't need to add all aliases. Let's keep the more descriptive version only.

**Pull request merged**